packages/ilios-common/addon/modifiers/date-picker.js

```diff
@@ -24,29 +28,49 @@ export default class DatePickerModifier extends Modifier {
     });
   }
 
-  modify(element, [value], { minDate, maxDate, locale, onChangeHandler }) {
+  modify(element, [value], { minDate, maxDate, allowedWeekdays, locale, onChangeHandler }) {
     // We only need to set this once.
     if (!this.onChangeHandler) {
       this.onChangeHandler = onChangeHandler;
     }
-    if (!this.flatpickr) {
-      this.locale = locale ?? this.intl.primaryLocale;
-      this.flatpickr = this.initPicker(element, value, minDate, maxDate, this.locale);
-    }
 
-    if (this.flatpickr.selectedDates[0] !== value) {
-      this.flatpickr.setDate(value);
-    }
-    if (this.flatpickr.minDate !== minDate) {
-      this.flatpickr.set('minDate', minDate);
-    }
-    if (this.flatpickr.maxDate !== maxDate) {
-      this.flatpickr.set('maxDate', maxDate);
+    if (Array.isArray(allowedWeekdays)) {
+      this.allowedWeekdays = allowedWeekdays;
```

Comment:
Edge case, but if we switch from a list to null passed in value this won't reset back to an empty array.

Reply:
fair point. i'll put an `else`  clause in, and default to empty array for anything else but array input.

Reply:
handled :heavy_check_mark: